pointerevents/pointerevent_boundary_events_in_capturing-manual.html

```diff
@@ -0,0 +1,97 @@
+              WaitingForSecondMove: "secondMove",
+              WaitingForUp:         "up"
+            }
+            var phase = PhaseEnum.WaitingForDown;
+            var eventsRecieved = [];
+
+            function resetTestState() {
+                eventsRecieved = [];
+                phase = PhaseEnum.WaitingForDown;
+            }
+            function run() {
+                var test_pointerEvent = setup_pointerevent_test("pointerevent boundary events in capturing", ALL_POINTERS);
+                var target = document.getElementById("target0");
+                var listener = document.getElementById("listener");
+
+                eventList.forEach(function(eventName) {
+                    on_event(target, eventName, function (event) {
+                        if (phase == PhaseEnum.WaitingForDown) {
+                            if (eventName == 'pointerdown') {
+                              listener.setPointerCapture(event.pointerId);
+                              phase = PhaseEnum.WaitingForFirstMove;
+                            }
+                        } else if (phase == PhaseEnum.WaitingForUp) {
+                            if (event.type == 'pointerup')
+                                test_pointerEvent.done();
+                        } else {
+                            eventsRecieved.push(event.type + '@target');
+                            if (phase == PhaseEnum.WaitingForSecondMove && event.type == 'pointermove') {
+                                test(function () {
+                                    checkPointerEventType(event);
+                                    assert_array_equals(eventsRecieved, ['lostpointercapture@listener', 'pointerout@listener', 'pointerleave@listener', 'pointerover@target', 'pointerenter@target', 'pointermove@target'],
+                                        'lostpointercapture and pointerout/leave should be dispatched to the capturing target and pointerover/enter should be dispatched to the hit-test element before the first pointermove event after releasing pointer capture');
+                                }, expectedPointerType + " pointer events boundary events when releasing capture"); 
+                                phase = PhaseEnum.WaitingForUp;
+                            }
+                        }
+                    });
+                    on_event(listener, eventName, function (event) {
+                        if (phase == PhaseEnum.WaitingForDown)
+                          return;
+                        eventsRecieved.push(event.type + '@listener');
+                        if (eventName == 'pointermove') {
+                          test(function () {
+                              checkPointerEventType(event);
+                              assert_array_equals(eventsRecieved, ['pointerout@target', 'pointerleave@target', 'pointerover@listener', 'pointerenter@listener', 'gotpointercapture@listener', 'pointermove@listener'],
```

Comment:
The only reason was that I didn't want to add the logic to suppress the next move events which is one solution. The other solution would be not to add any pointermove at all in the array so that it doesn't make the array equal logic any more complicated.